packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts

```diff
@@ -74,7 +74,8 @@ export const topicNameLabelStyle = mergeStyles({
  */
 export const participantListWrapper = mergeStyles({
   boxShadow: '0px 0.3px 0.9px rgba(0, 0, 0, 0.1), 0px 1.6px 3.6px rgba(0, 0, 0, 0.13)',
-  width: '15rem',
+  width: '20rem',
+  maxWidth: '50vw',
```

Comment:
Please add a comment above this line to say that the max width ensures that the message thread has enough space on narrow viewport.

Reply:
Issue with using `vw` is our composite should be independant of the viewport size - but instead be dependant on the parent container width - i.e. we don't know our composite will take up 100% of the viewport. If someone makes our composite 50% of the viewport size I think this will mean the participantListWrapper will be 100% of the composite width.

Can we use `width: 50%` here?